Source/Cogworks.Umbraco.Essentials/Extensions/ImageCropperExtensions.cs

```diff
@@ -34,20 +34,34 @@ public static string GetCropUrls(this IPublishedContent image, string cropAlias,
                 : imageUrl;
         }
 
-        private static string UpdateCropDimensions(string imageCrop, int? width, int? height)
+        private static string UpdateCropSettings(string imageCrop, int? width, int? height, bool enableWebP, int? quality = null)
         {
             var query = HttpUtility.ParseQueryString(imageCrop);
 
             if (width.HasValue)
             {
+                query.Remove(ImageCropConstants.Width);
```

Comment:
Is this necessairy to remove this?

Reply:
I don't think it is necessary but it is probably safer